pywavefront/parser.py

```diff
@@ -47,21 +47,19 @@ def read_file(self, file_name):
     def parse(self, line, dir):
         """Determine what type of line we are and dispatch
         appropriately."""
-        if line.startswith('#'.encode("utf-8")):
```

Comment:
This was a change for Python 3:

https://github.com/greenmoss/PyWavefront/commit/3c747cab7fddb598ef54356c2f9fb247dc812846#diff-152490cb77ff65f57c586245c34af0edR55

Need to find a solution that works with both.